### PR TITLE
Readme updated to explicitly specify the changes required in DBCon.py file

### PR DESCRIPTION
**Pull request by aakashbansal**

### CODE REVIEW

**Comment by vivonk** [comment]
actually, if someone is using the environment variables for the DB connection then don't need to change. So specify it that if you are not using environment variables then use something like this. 
All the way Nice job :+1:

**Comment by aakashbansal** [comment]
Thanks for the suggestion. I have updated the readme to take into account the environment variables configuration.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,17 @@ mysql -u root -p  Bassa < Bassa.sql
 
 3. Open components/core/DBCon.py and setup database username and password.
 
+The required changes in DBCon.py would look like :
+
+If the environment variables are being used, modify as following :
+```
+_db=MySQLdb.connect("db", os.environ.get('YOUR_DB_USERNAME'), os.environ.get('YOUR_DB_PASSWORD'), os.environ.get('Bassa'))
```
Comment (agentmilindu) [inline README.md]: Shall we change `os.environ.get('Bassa')` into `os.environ.get('YOUR_DB_NAME')`?

**Comment by kmehant** [comment]
@aakashbansal closing this as it is fixed now 😄

**Pull request closed without merging**